### PR TITLE
Chore: RabbitMQ dockercompose

### PR DESCRIPTION
**Pull request by guyzsarun**
Added: Docker-compose for all RabbitMQ components

### CODE REVIEW

File: rabbitmq/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    environment:
+      SPRING_RABBITMQ_HOST: rabbitmq
+      SPRING_DATASOURCE_URL: jdbc:postgresql://db:5432/postgres
+  db:
+    image: postgres:14.0-alpine
+    restart: always
+    ports:
+      - 5432:5432
+    environment:
+      POSTGRES_PASSWORD: 123
```
Comment (Raksani) [inline rabbitmq/docker-compose.yml:29]: How about USER and DB name? shall we add those to this file as well? 
```
POSTGRES_USER: postgres
```

File: rabbitmq/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      - 8080:8080
+    environment:
+      SPRING_RABBITMQ_HOST: rabbitmq
+  spring-consumer:
+    build: spring-rabbitmq-consumer
+    ports:
+      - 8081:8081
+    environment:
+      SPRING_RABBITMQ_HOST: rabbitmq
+      SPRING_DATASOURCE_URL: jdbc:postgresql://db:5432/postgres
```
Comment (Raksani) [inline rabbitmq/docker-compose.yml]:
```suggestion
      SPRING_DATASOURCE_URL: jdbc:postgresql://db:5432/testdb
```

File: rabbitmq/spring-rabbitmq-consumer/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+WORKDIR /build/
+RUN mvn package
+
+FROM openjdk:8-jre-alpine
+
+WORKDIR /app
+
+COPY --from=MAVEN_BUILD /build/target/spring-rabbitmq-consumer-0.0.1-SNAPSHOT.jar /app/
+
+ENTRYPOINT ["java", "-jar", "spring-rabbitmq-consumer-0.0.1-SNAPSHOT.jar"]
```
Comment (Raksani) [inline rabbitmq/spring-rabbitmq-consumer/Dockerfile:15]:
```suggestion
ENTRYPOINT ["java", "-jar", "spring-rabbitmq-consumer-0.0.1-SNAPSHOT.jar"]

```

File: rabbitmq/spring-rabbitmq-producer/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+WORKDIR /build/
+RUN mvn package
+
+FROM openjdk:8-jre-alpine
+
+WORKDIR /app
+
+COPY --from=MAVEN_BUILD /build/target/spring-rabbitmq-producer-0.0.1-SNAPSHOT.jar /app/
+
+ENTRYPOINT ["java", "-jar", "spring-rabbitmq-producer-0.0.1-SNAPSHOT.jar"]
```
Comment (Raksani) [inline rabbitmq/spring-rabbitmq-producer/Dockerfile:15]:
```suggestion
ENTRYPOINT ["java", "-jar", "spring-rabbitmq-producer-0.0.1-SNAPSHOT.jar"]

```

**Review by Raksani — CHANGES_REQUESTED** [review]

File: rabbitmq/docker-compose.yml (hunk 1 above)
Comment (guyzsarun) [reply rabbitmq/docker-compose.yml:29]: Done

**Pull request merged**